app/_data/entity_examples/certificates/example-certificate.yml

```diff
@@ -0,0 +1,53 @@
+cert: |
```

Comment:
I got the certificate and private key example from https://docs.vmware.com/en/VMware-NSX-Data-Center-for-vSphere/6.4/com.vmware.nsx.admin.doc/GUID-BBC4804F-AC54-4DD2-BF6B-ECD2F60083F6.html. Let me know if we should use something else.